### ISSUE TITLE
Less support of CSS compilation is yelling an error

### ISSUE THREAD

**Issue by gkorland** [acdvorak/intellij-lessc-plugin#54]
Using the following line which is legal with LESS to direct the compiler to compile CSS as LESS file, shows an error in the IDE.

``` CSS
@import (less) '/js/libs/jquery-ui/themes/smoothness/jquery-ui.min.css';
```

**Comment by dannymachal** [comment]
This feature I believe wasn't implemented until LESS v1.4. Since this plugin is still using LESS 1.3.x that is probably why.

From http://lesscss.org/

In v1.4.0 you can force a file to be interpreted as a particular type by specifying an option, e.g.

@import (css) "lib";
Will output..

@import "lib";
and

@import (less) "lib.css";
Will import the lib.css file and treat it as less. If you specify a file is less and do not include an extension, none will be added.

**Comment by gkorland** [comment]
Right, are there any plans to upgrade the support to 1.4?

**Comment by dannymachal** [comment]
I would hope so. I hate having to run something like WinLess all the time when PHPstorm is my go to IDE. I spent the morning trying to "update it." Then I realized I had no idea what I was doing and I don't know any Java, how to use Maven properly, or how to use/configure IntelliJ IDEA for making plugins. At least I figured out those were the things I needed to learn to make this work. 

I thought I would be crafty and drop in the less-rhino-1.4.0.js and change some code to reflect this and pray it would build. All my willy-nilly-options and configurations did get something to build. But it wasn't anything like the Plugin Package PHPstorm wants. So maybe I'll keep at it.
